index.html

```diff
@@ -76,5 +74,168 @@ <h3><a class="connect" href="#">LET'S CONNECT</a></h3>
         </div>
       </div>
     </section>
+
+    <!-- Works Section -->
+    <section class="works">
+      <div class="cards">
+        <div class="project1">
+          <img
+            class="picture"
+            src="Images/Snapshoot Portfolio.png"
+            alt="Snapshoot Portfolio Pic 1"
+          />
+          <h1 class="project-title">Tonic</h1>
+          <ul class="project-summary">
+            <li><h3 class="canopy">CANOPY</h3></li>
+            <li><h3 class="back-end-dev">Back End Dev</h3></li>
+            <li><h3 class="year">2015</h3></li>
+          </ul>
+          <p class="project-details">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <div class="buttons">
+            <button type="button">html</button></li>
+            <button type="button">css</button>
+            <button type="button">javaScript</button>
+          </div>
+          <button class="project-button" type="button">See Project</button>
+        </div>
+        </div>
+        <div class="cards">
+        <div class="project2">
+          <img
+            class="picture"
+            src="Images/Snapshoot Portfolio (1).png"
+            alt="Snapshoot Portfolio Pic 2"
+          />
+          <h1 class="project-title">Multi-Post Stories</h1>
+          <ul class="project-summary">
+            <li><h3 class="canopy">CANOPY</h3></li>
+            <li><h3 class="back-end-dev">Back End Dev</h3></li>
+            <li><h3 class="year">2015</h3></li>
+          </ul>
```

Comment:
Kindly include the bullets for the stack and year details of the project as shown on the design.

Your design:
![image](https://user-images.githubusercontent.com/20554583/206015425-6c729388-9f98-478a-87dc-14b62b7a60b6.png)

Figma design:
![image](https://user-images.githubusercontent.com/20554583/206015458-493502a8-1cdf-4473-88c1-6d7e300e6a86.png)